packages/components/styles/_animation.scss

```diff
@@ -0,0 +1,16 @@
+@import "./animations/vendor/util/args";
```

Comment:
Are you actually using any of this? I can't see any usages

Reply:
Soon™ (there might be other components that need it...)